core/modules/replica/HttpIngestConfigModule.cc

```diff
@@ -87,6 +87,12 @@ json HttpIngestConfigModule::_get() {
                     databaseInfo.name, HttpFileReaderConfig::category, key).value);
         } catch (DatabaseServicesNotFound const&) {}
     };
+    auto const getLong = [&databaseServices, &databaseInfo](json& obj, string const& key) {
+        try {
+            obj[key] = stol(databaseServices->ingestParam(
+                    databaseInfo.name, HttpFileReaderConfig::category, key).value);
+        } catch (DatabaseServicesNotFound const&) {}
+    };
```

Comment:
There is a lot of duplicated code in getInt/getLong/getStr methods, feels like it wants to be a templated method somewhere. Same for updateInt/updateLong/updateStr functions below.